### DISCUSSION TITLE
fzf Parameters?

### DISCUSSION THREAD

**Discussion by bertradio** [sxyazi/yazi#878] [1. Q&A]
I would like to add some parameters to fzf using the jump command bound to Z. Is there a way to do that? Or accomplish it some other way? In particular, I'd like to add -e and --reverse to the call to fzf.

SOLUTION. If I export FZF_DEFAULT_OPTS in bash with a string of options it works. So probably don't need to specify separatey n Yazi

**Comment by sxyazi** [comment]
Nice to see your issue has been resolved.

I'll reimplement fzf as a built-in plugin in the next few days to provide greater flexibility. I've just merged zoxide for this, https://github.com/sxyazi/yazi/issues/865.

**Comment by bertradio** [comment]
I see that you have re-implemented zoxide as a plugin. I can't find it. How do I add this. I have compiled the latest version, but I get a message that zoxide has been reimplement.

I love yazi. Great work! Thenks.

**Comment by bertradio** [comment]
I was looking to download a plugin. But it seems to be part of the app. So I changed my keymap.toml with plugin zoxide and it seems to work.

**Reply by sxyazi** [reply]
Yep, it's a built-in plugin, so you don't need to manually download it. I've finished the work of implementing `fzf` as a plugin in https://github.com/sxyazi/yazi/pull/884

**Comment by bertradio** [comment]
Fzf works well. Is there a way to add options to fzf?
